src/core/app/types.ts

```diff
@@ -76,24 +77,28 @@ export interface AppConfigGlobal {
 }
 
 // Unified config from /app/configuration
-export type AppConfiguration = {
+export interface AppDto {
```

Comment:
```suggestion
export interface AppConfigurationDto {
```

Reply:
It's name of BE schema, we can review that in config refactor task